core/modules/macros/contrastcolour.js

```diff
@@ -29,6 +29,22 @@ exports.params = [
 Run the macro
 */
 exports.run = function(target,fallbackTarget,colourA,colourB) {
```

Comment:
I'm afraid I don't see the purpose of these changes to the contrastcolour macro. Can you describe the change you're making more clearly?